config/locales/client.en.yml

```diff
@@ -3136,6 +3136,10 @@ en:
     safe_mode:
       enabled: "Safe mode is enabled, to exit safe mode close this browser window"
 
+    too_many_sessions:
+      one: You were logged in to more than %{limit} devices. The oldest %{count} session has been logged out. Review recently used devices in your <a href="%{url}">account preferences</a>.
+      other: You were logged in to more than %{limit} devices. The oldest %{count} sessions have been logged out. Review recently used devices in your <a href="%{url}">account preferences</a>.
```

Comment:
Those translations are kinda problematic as they contain 2 numbers. I recommend using [Message Format](https://meta.discourse.org/t/message-format-support-for-localization/7035), because the alternative of splitting the string into two parts isn't great either.

Reply:
I did think about that, but I decided `limit` was a constant equal to 100 (or a similarly high number), so there is never a need for singular/plural on that variable. Do you think we still need to use message format? Do some languages have different grammar even for higher numbers?